commands/utility/infractions.js

```diff
@@ -0,0 +1,87 @@
+const Discord = require('discord.js');
+
+module.exports = {
+	name: 'infractions',
+	description: 'finds user infraction record in db and returns it to channel',
+	execute(msg,con) {
+		// Make sure only SU, Mods and Admin can run the command
+		const targetUser = msg.mentions.members.first();
+		if (canWarn(msg)){
```

Comment:
I think it'd be better to rename this to something like `canUse()` or `canCheckInfractions()`.

Reply:
Fixed